bin/tictactoe

```diff
@@ -1,3 +1,45 @@
 #!/usr/bin/env ruby
-
 require_relative '../config/environment'
+
+
+def choose_game
```

Comment:
@johnfewell I think abstracting lines 5 - 45 into a CLI class in the lib folder would be the better way to do this. The application could then just call 

```CLI.new.choose_game```

on line 5 and end of file